docs/search/operations.md

```diff
@@ -3,34 +3,33 @@ sidebar_position: 40
 title: Search Queries
 ---
 
-The `SEARCH` keyword starts the language construct to filter Views of type Search. Conceptually, a View is just another document data source, similar to an array or a document/edge collection, over which you can iterate using a [FOR operation](../queries/c8ql/operations/for.md) in C8QL:
+You can use the `SEARCH` keyword to filter a query using a search view, allowing you to:
 
-```sql
-FOR doc IN viewName
-  RETURN doc
-```
+- Filter documents based on C8QL Boolean expressions and functions.
+- Match documents located in different collections.
+- Sort the result set based on how closely each document matches the search conditions.
 
-The optional `SEARCH` operation provides the capabilities to:
+Refer to [Search Views](/search/views/index.md) for information about creating a custom search view.
 
-- filter documents based on C8QL Boolean expressions and functions
-- match documents located in different collections backed by a fast index
-- sort the result set based on how closely each document matched the search conditions
+## Syntax
 
-See [Search Views](../index.md) on how to set up a View.
-
-## General Syntax
-
-The `SEARCH` keyword is followed by an Search filter expressions, which is mostly comprised of calls to Search C8QL functions.
+You must use the `SEARCH` statement in a `FOR...IN` operation. The `SEARCH` statement must be placed after the `FOR` and before any additional operations.
 
 ```sql
-FOR doc IN viewName
-  SEARCH expression OPTIONS {…}
+FOR doc IN <SEARCH_VIEW_NAME>
+  SEARCH <EXPRESSION> OPTIONS {…}
   ...
 ```
 
-The `SEARCH` statement, in contrast to `FILTER`, is treated as a part of the `FOR` operation, not as an individual statement. It can not be placed freely in a query nor multiple times in the body of a `FOR` loop. `FOR ... IN` must be followed by the name of a View, not a collection. The `SEARCH` operation has to follow next, other operations before `SEARCH` such as `FILTER`, `COLLECT` etc. are not allowed in this position. Subsequent operations are possible after `SEARCH` and the expression however, including `SORT` to order the search results based on a ranking value computed by the Search View.
+For example:
+
+```sql
+FOR doc IN MySearchView
+  SEARCH ANALYZER(doc.text == "quick" OR doc.text == "brown", "text_en")
+RETURN doc
+```
 
-`expression` must be an Search expression. The full power of Search is harnessed and exposed via special [Search functions](functions.md), during both the search and sort stages. On top of that, common C8QL operators are supported:
+Replace `SEARCH_VIEW_NAME` with the name of your search view, and `EXPRESSION` with one of the following [Search functions](functions.md):
 
```

Comment:
AND, OR ... are logical and comparison operators; they are not Search functions. PHRASE STARTSWITH() ... are Search functions